python/tvm/relay/frontend/pytorch.py

```diff
@@ -399,7 +399,10 @@ def squeeze(self, inputs, input_types):
             axis = None
         else:
             # TODO (t-vi): why is the cast to int needed? similarly elsewhere
-            axis = [int(inputs[1])]
+            # [MBS] squeeze axis can be a list
+            # axis = [int(inputs[1])]
```

Comment:
Probably the old code can be removed?

Reply:
yes, removed.

Reply:
test added with other tests for this frontend.